fsb/src/lib.rs

```diff
@@ -0,0 +1,21 @@
+#![allow(non_snake_case)]
+#[allow(dead_code)]
+#[macro_use]
```

Comment:
Is it truly necessary to allow dead code? It also would be nice to add crate-level docs, `no_std` and other attributes (see other crates for reference).

Reply:
used `no_std`, and I depend on `alloc` now for dynamically sized arrays. We use them [here](https://github.com/iquerejeta/hashes/blob/master/fsb/src/macros.rs#L37).